### ISSUE TITLE
Navigation buttons don't work

### ISSUE THREAD

**Issue by namr33b** [overtake/TelegramSwift#393]
Buttons 'page-up', 'page-down', 'home' and 'end' do nothing. Scrolling big groups just by mouse not very comfortably.

**Comment by Scvairy** [comment]
I think that you mean `Fn+←` as `home`, `Fn+↑` as `page-up`. But you are right.

**Comment by namr33b** [comment]
I have full size keyboard, with separate buttons :)

**Comment by Scvairy** [comment]
Non-apple keyboard? I didn't see Apple keyboards with such keys. But on Apple keyboards, they can be used by what I mentioned above.

**Comment by namr33b** [comment]
For example https://www.apple.com/shop/product/MQ052LL/A/magic-keyboard-with-numeric-keypad-us-english-silver

**Comment by Scvairy** [comment]
Okay, you got me ;)

**Comment by MacDragon** [comment]
This, why can I not page up/down through chat text?

**Comment by theirix** [comment]
Fn+Up / Fn+Down does not scroll chat view at all, even with a laptop keyboard. Only trackpad swipes work. It is very uncomfortable.

**Comment by BATCOH** [comment]
I use the home and end buttons to move the cursor to the beginning or end of a line when writing messages.
This works everywhere on mac except Telegram for mac.
Until this bug is fixed - I made a [little modification for Karabiner Elements](https://gist.github.com/BATCOH/f8b74b897c3f00c712d155acd00b0bfc) that remaps "home" to ⌘← and "end" to ⌘→ when Telegram is frontmost application.

**Comment by tmojlupo** [comment]
This is still an issue in 9.1.1.238861.

**Comment by zv09** [comment]
still the same
PageUp/PageDown  is not reacting for groups/chats scrolling 
it works before, but I don't remember a ver of app it was good.. 
MacBookPro native model, all the keyboard and keys are from Apple
